Dockerfile

```diff
@@ -18,4 +18,4 @@ COPY subprojects /app/subprojects
 COPY test /app/test
```

Comment:
Can you copy over the tools directory as well, it has utils in there required for the rpc to work

Reply:
nevermind i'm crazy it's in src/